themes/color-blue-gray.yaml

```diff
@@ -32,17 +80,23 @@ global:
     # global to "" (null).
     sdk_icon_color_style: colorized_icons
 
+    sdk_terrain_shading_earth_style: terrain-shading-earth
```

Comment:
Whats the reason of having these globals defined in each color-* instead of just defining these in terrain-shading*?

Reply:
because the sphere map gets inverted for the dark themes and it needs to switch

Reply:
I see, makes sense.

Reply:
here's where it switches in inverted theme:

https://github.com/tangrams/refill-style/blob/sensescape/texture-relief/themes/color-inverted.yaml#L72-L74